frontend/src/components/Account/MyAccount/index.js

```diff
@@ -6,16 +6,20 @@ import ProfileApi from "../../../api/profileApi";
 import "./MyAccount.scss";
 
 export const MyAccount = () => {
-  const [userProfile, setUserProfile] = useState();
+  const [userProfile, setUserProfile] = useState(null);
   const [familyMembers, setFamilyMembers] = useState();
 
-  useEffect(() => {
+  const getUserDetails = () => {
     ProfileApi.get().then(setUserProfile);
-  }, []);
+  };
 
-  useEffect(() => {
+  const getUserDependantsDetails = () => {
     ProfileApi.getDependants().then(setFamilyMembers);
-  }, []);
+  };
+
+  useEffect(() => getUserDetails(), []);
+
+  useEffect(() => getUserDependantsDetails(), []);
```

Comment:
We could use one effect here.
```suggestion
  useEffect(() => {
    getUserDetails();
    getUserDependantsDetails()
  }, []);
```

Reply:
I just encapsulated the existing code in some functions. Only one effect is needed, indeed.